cmd/ocitools/generate.go

```diff
@@ -382,3 +406,15 @@ func setupLinuxNamespaces(g generate.Generator, needsNewUser bool, nsMaps map[st
 		g.AddOrReplaceLinuxNamespace(nsName, nsPath)
 	}
 }
+
+func onlyExportFlagSpecified(context *cli.Context) bool {
+	onlyExportFlags := []bool{
+		context.Bool("seccomp-only"),
+	}
+	for _, flag := range onlyExportFlags {
+		if flag {
+			return true
```

Comment:
On Thu, Jul 21, 2016 at 01:22:06PM -0700, Grant Seltzer Richman wrote:
> > +func onlyExportFlagSpecified(context *cli.Context) bool {
> > +	onlyExportFlags := []bool{
> > +		context.Bool("seccomp-only"),
> > +	}
> > +	for _, flag := range onlyExportFlags {
> > +		if flag {
> > +			return true
> 
> I was saying so that we can add functionality for the same way
> `seccomp-only` would work

But the way seccomp-only works in 3a5ad1e7 is by bypassing the usual
Save logic and rolling its own.  I'd rather have it work by teaching
the usual Save logic about --*-only.